### ISSUE TITLE
Please consider using simple auth to avoid going through 

### ISSUE THREAD

**Issue by vikramkhatri** [IBMStockTrader/tradr#1]
https://github.com/IBMStockTrader/tradr/blob/f7cb286623ccf43ac6774d69b2140d5f4387cf4e/server.js#L42

John, 

Is it possible to use simple auth for the demo purpose like stock / trader you have for trader app? And, those are switchable through some param as some would still like to see openid auth.

Thank you.

**Comment by greghint** [comment]
Yes, we should default to the simple stock/trader login just like the `trader` UI.

**Comment by jwalcorn** [comment]
Perhaps not exactly what's requested, but we do have Stock Trader working with a local KeyCloak, which is the upstream open source project that the Red Hat Single Sign-On (RH-SSO) product is based on.  So if you deploy a KeyCloak (pretty easy, when using its operator) and define ids/pwds there, you can configure Tradr to authenticate against that OIDC provider, rather than having to use something out on the cloud like IBMid.  You could even get fancy and back that KeyCloak by an LDAP (such as BluePages), if you wanted.
